### DISCUSSION TITLE
Theme - Provide other themes or theming support

### DISCUSSION THREAD

**Discussion by markus-ruderman** [solidtime-io/solidtime#286] [Feature requests]
Thanks for this great Clockify alternative 😃. I'm still testing the latest version and do some comparisons and I detected that the current dark theme hurts my eye's a little bit 😅 (it has to much contrast) and would be great to get the possibility for other themes, maybe similar look like Clockify.

I tried out to modify the color & font directly via the browser Dev Tools and here is my result - I have crossed out sensitive data (red).

Here the original:

![image](https://github.com/user-attachments/assets/f248546e-ae6b-4bbb-b790-a6b8ff563f17)

Here is the adapted (bgcolor & font-family) one similar direction as Clockify - only to show the direction:

![image](https://github.com/user-attachments/assets/6b01e1b2-16b6-4399-b635-8e8524f8e0ab)

**Comment by korridor** [comment]
Thanks for the suggestion! I converted this issue to a feature request discussion.

**Comment by markus-ruderman** [comment]
A similar discussion is also **Light theme** #234.
